cargo-afl/src/bin/cargo-afl.rs

```diff
@@ -343,6 +376,9 @@ where
         .env("RUSTDOCFLAGS", &rustdocflags)
         .env("ASAN_OPTIONS", asan_options)
         .env("TSAN_OPTIONS", tsan_options)
+        .env("AFL_LLVM_INSTRUMENT", "PCGUARD")
+        .env("AFL_LLVM_CMPLOG", "1")
+        .env("AFL_QUIET", "1")
```

Comment:
Shouldn't these be guarded by the `cmplog` feature?

If that's correct, then I would set these in the same `if` block above where the `-Z` flags were added.